cvat-ui/src/components/labels-editor/label-form.tsx

```diff
@@ -150,7 +171,7 @@ export default class LabelForm extends React.Component<Props> {
         const { key } = fieldInstance;
         const attrNames = this.formRef.current?.getFieldValue('attributes')
             .filter((_attr: any) => _attr.id !== attr.id).map((_attr: any) => _attr.name);
-
+        const height = window.innerWidth < 1404 ? 2 : 1;
```

Comment:
Why do we need this? It causes differences in heights in between elements.

Reply:
THe thing is when we change the width below 1404px, the 'name' moves to next line which can't be displayed. So in order to see that we need to increase the height.
[Screencast from 16-07-24 02:19:30 PM IST.webm](https://github.com/user-attachments/assets/a489a90d-d860-4560-95a4-c13efe8e83d4)